packages/asset-service/src/generateAssetData/ethTokens/ethereum.ts

```diff
@@ -19,22 +20,31 @@ import {
 
 export const addTokensToEth = async (): Promise<BaseAsset> => {
   const baseAsset = ethereum
-  const [ethTokens, yearnVaults, ironBankTokens, zapperTokens, underlyingTokens, foxyToken] =
-    await Promise.all([
-      getUniswapTokens(),
-      getYearnVaults(),
-      getIronBankTokens(),
-      getZapperTokens(),
-      getUnderlyingVaultTokens(),
-      getFoxyToken()
-    ])
+  const [
+    ethTokens,
+    yearnVaults,
+    ironBankTokens,
+    zapperTokens,
+    underlyingTokens,
+    foxyToken,
+    uniV2Token
+  ] = await Promise.all([
+    getUniswapTokens(),
+    getYearnVaults(),
+    getIronBankTokens(),
+    getZapperTokens(),
+    getUnderlyingVaultTokens(),
+    getFoxyToken(),
+    getUniV2Token()
```

Comment:
I don't understand why we have synchronous functions in here, but it was approved for FOXy, so... 🤷‍♂️